README.adoc

```diff
@@ -48,11 +48,12 @@ Please, complement the content of this readme document with the link:https://git
 Make sure you have the following tools:
 
 * The link:http://golang.org/doc/install[Go Programming Language]
-** Currently, Kiali releases are built using Go 1.16. Although Kiali may build correctly using other versions of Go, it's suggested to use version 1.16.2 for development to ensure replicatable builds. Makefiles will require this minimum version of Go.
+** Currently, Kiali releases are built using Go 1.17. Although Kiali may build correctly using other versions of Go, it's suggested to use version 1.17.7 for development to ensure replicatable builds. Makefiles will require this minimum version of Go.
```

Comment:
Rather than continuing to hardcode the versions here (and going obsolete after we bump the go but forget to update this readme :), I suggest pointing the reader to the Makefile - specifically here: https://github.com/kiali/kiali/blob/master/Makefile#L31

So I suggest changing this to:
```suggestion
** Currently, Kiali releases are built using a specific minimum version of Go as declared in the [Makefile](https://github.com/kiali/kiali/blob/master/Makefile#L31). Although Kiali may build correctly using other versions of Go, it's suggested to use the version that the Makefile uses for your own development to ensure replicatable builds.
```